src/particles.cpp

```diff
@@ -71,11 +77,24 @@ Particle::Particle(
 	m_material.setFlag(video::EMF_BACK_FACE_CULLING, false);
 	m_material.setFlag(video::EMF_BILINEAR_FILTER, false);
 	m_material.setFlag(video::EMF_FOG_ENABLE, true);
-	m_material.MaterialType = video::EMT_TRANSPARENT_ALPHA_CHANNEL;
+	if (material_type_param != 8464) {
```

Comment:
The best way would be to use a define and put a comment on top of it to describe how you calculated it.